test/weighting/README

```diff
@@ -0,0 +1,17 @@
+
+Weighting Test-case
+===========================================
+
+This test uses the various aerosol weighting types on a single
+scenario to show that the results are the same between all the 
+different options. There are the following configurations:
+
+- run_part.spec: source class (n_class=3) with number and mass weighting (n_group=2)
```

Comment:
This default one is currently `nummass_source`, I believe? Would it be clearer to make them all explicit, so this would be `run_part_nummass_source.spec` and it would explicitly specify the weighting type inside the spec file?

Also, we have plain `flat` but we are missing plain `nummass`?

We are also missing all the `power` variants but I think that's ok.

Reply:
Made more clear by renaming default to more clearly be `nummass_source` and specify the weighting rather than call it default. Added `nummass` as an additional test.